### PR TITLE
[DOCS][OV JS] Update JS API readme

### PR DESCRIPTION
**Pull request by vishniakov-nikolai**
### Details:
 - Actualize JS API readme file for npm
 - Fix link to node directory
 - Fix .npmignore to do not include extra files into npm package

### CODE REVIEW

File: src/bindings/js/docs/README.md [hunk 1]
```diff
@@ -1,4 +1,57 @@
...
+
+- [include](../node/include/) - header files for current API.
+- [lib](../node/lib/) - TypeScript sources for current API.
+- [src](../node/src/) - C++ sources for current API.
+- [tests](../node/tests/) - tests directory for current API.
+
+### Build
+
+- Make sure that all submodules are updated `git submodule update --init --recursive`
+- Create build dir `mkdir build && cd build`
```
Comment (sgolebiewski-intel) [inline src/bindings/js/docs/README.md:20]:
````suggestion
- Create the *build* directory:
   ```
   mkdir build && cd build
   ```
````

File: src/bindings/js/docs/README.md [hunk 2]
```diff
@@ -1,4 +1,57 @@
...
+### Components
+
+- [include](../node/include/) - header files for current API.
+- [lib](../node/lib/) - TypeScript sources for current API.
+- [src](../node/src/) - C++ sources for current API.
+- [tests](../node/tests/) - tests directory for current API.
+
+### Build
+
+- Make sure that all submodules are updated `git submodule update --init --recursive`
```
Comment (sgolebiewski-intel) [inline src/bindings/js/docs/README.md:19]:
````suggestion
- Make sure that all submodules are updated:
  ```
  git submodule update --init --recursive
  ```
````

File: src/bindings/js/docs/README.md [hunk 3]
```diff
@@ -1,4 +1,57 @@
...
+- Create build dir `mkdir build && cd build`
+- Configure binaries building:
```
Comment (sgolebiewski-intel) [inline src/bindings/js/docs/README.md:21]:
```suggestion
- Configure building of the binaries:
```

File: src/bindings/js/docs/README.md [hunk 4]
```diff
@@ -1,4 +1,57 @@
...
+    -DENABLE_WHEEL=OFF \
+    -DENABLE_PYTHON=OFF \
+    -DENABLE_INTEL_GPU=OFF \
+    -DCMAKE_INSTALL_PREFIX=../src/bindings/js/node/bin \
+    ..
+  ```
+- Build bindings:
+  `cmake --build . --config Release --verbose -j4`
+- Install binaries for openvino-node package:
+  `cmake --install .`
```
Comment (sgolebiewski-intel) [inline src/bindings/js/docs/README.md:39]:
````suggestion
- Install binaries for the *openvino-node* package:
  ```bash
  cmake --install .
  ```
````

File: src/bindings/js/docs/README.md [hunk 5]
```diff
@@ -1,4 +1,57 @@
...
+    -DENABLE_TESTS=OFF \
+    -DENABLE_SAMPLES=OFF \
+    -DENABLE_WHEEL=OFF \
+    -DENABLE_PYTHON=OFF \
+    -DENABLE_INTEL_GPU=OFF \
+    -DCMAKE_INSTALL_PREFIX=../src/bindings/js/node/bin \
+    ..
+  ```
+- Build bindings:
+  `cmake --build . --config Release --verbose -j4`
```
[the remark this answers is not in the document]
Comment (vishniakov-nikolai) [reply src/bindings/js/docs/README.md:43]: Fix build instruction

File: src/bindings/js/docs/README.md (hunk 4 above)
Comment (vishniakov-nikolai) [reply src/bindings/js/docs/README.md:45]: Fix install instruction

File: src/bindings/js/docs/README.md [hunk 6]
```diff
@@ -1,4 +1,57 @@
...
+- Install binaries for openvino-node package:
+  `cmake --install .`
+- Go to npm package folder `cd ../src/bindings/js/node`
+- Now you can install dependencies packages and transpile ts to js code. Run `npm install`
+- Run tests `npm run test` to make sure that **openvino-node** built successfully
+
+## Usage
+
+- Add `openvino-node` package in your project, specify in **package.json**: `"openvino-node": "file:*path-to-current-directory*"`
+- Require by: `const ov = require('openvino-node');`
```
Comment (sgolebiewski-intel) [inline src/bindings/js/docs/README.md:56]:
````suggestion
- Add the **openvino-node** package to your project by specifying it in **package.json**: 
  ```json
  "openvino-node": "file:*path-to-current-directory*"
  ```
- Make sure to require it:
  ```
  const ov = require('openvino-node');
  ```
````

File: src/bindings/js/node/README.md [hunk 7]
```diff
@@ -1,50 +1,37 @@
...
-- Build bindings:
-  `cmake --build . --config Release --verbose -j4`
-- Install binaries for openvino-node package:
-  `cmake --install .`
-- Go to npm package folder `cd ../src/bindings/js/node`
-- Now you can install dependencies packages and transpile ts to js code. Run `npm install`
-- Run tests `npm run test` to make sure that **openvino-node** built successfully
+# OpenVINO™ Node.js Bindings
+
+Use OpenVINO Runtime from your Node.js application uses OpenVINO JavaScript API
```
Comment (sgolebiewski-intel) [inline src/bindings/js/node/README.md:3]:
```suggestion
Use OpenVINO JavaScript API for your Node.js application.
```
That still will be the same what you meant, right?

Comment (vishniakov-nikolai) [reply src/bindings/js/node/README.md:3]: It's different, but let's simplify this

File: src/bindings/js/node/README.md [hunk 8]
```diff
@@ -1,50 +1,37 @@
...
 
 ## Usage
 
-- Add `openvino-node` package in your project, specify in **package.json**: `"openvino-node": "file:*path-to-current-directory*"`
-- Require by: `const ov = require('openvino-node');`
+Install openvino-node package
```
Comment (sgolebiewski-intel) [inline src/bindings/js/node/README.md:7]:
```suggestion
Install the **openvino-node** package:
```

File: src/bindings/js/node/README.md [hunk 9]
```diff
@@ -1,50 +1,37 @@
...
+
+```sh
+npm install openvino-node
+```
+
+Use openvino-node package
```
Comment (sgolebiewski-intel) [inline src/bindings/js/node/README.md:13]:
```suggestion
Use the **openvino-node** package
```

File: src/bindings/js/node/README.md [hunk 10]
```diff
@@ -1,50 +1,37 @@
...
-## Samples
+## Build From Sources
 
-[Samples & notebooks of OpenVINO Node.js API](../../../../samples/js/node/README.md)
+For more details, refer to the [Developer documentation](https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/js/docs/README.md#openvino-node-package-developer-documentation)
+
+## Documentation & Samples
+
+- [OpenVINO Node.js API](https://docs.openvino.ai/2024/api/nodejs_api/nodejs_api.html)
+- [notebooks & samples openvino-node package usage](https://github.com/openvinotoolkit/openvino/blob/master/samples/js/node/README.md)
```
Comment (sgolebiewski-intel) [inline src/bindings/js/node/README.md:26]:
```suggestion
- [Samples of OpenVINO Node.js API](https://github.com/openvinotoolkit/openvino/blob/master/samples/js/node/README.md)
```

**Review by almilosz — APPROVED** [review]

**Pull request merged**